src/app/backend/handler/apihandler.go

```diff
@@ -584,10 +583,9 @@ func CreateHTTPAPIHandler(iManager integration.IntegrationManager, cManager clie
 
 	apiV1Ws.Route(
 		apiV1Ws.POST("/tenants/{tenant}/namespace"). // TODO
-								To(apiHandler.handleGetNamespacesWithMultiTenancy).
+								To(apiHandler.handleCreateNamespace).
```

Comment:
revert this function name to handleGetNamespacesWithMultiTenancy()

Reply:
The function name is as per Centaurus dashboard code and working optimally. Thanks.